project/task1/task1.py

```diff
@@ -0,0 +1,67 @@
+import math
+import csv
+import copy
+from pathlib import Path
+from pointClass import Point
+from detectFixations import detectFixations
+
+base_path = Path(__file__).parent
+file_path = (base_path / '../data/train.csv').resolve()
+
+# Load assigned data for group 7 to variable 'data'
+with file_path.open() as dataFile:
+    rawData = []
+    for row in csv.reader(dataFile, delimiter=','):
+        if (row[0] == 's7') or (row[0] == 's17') or (row[0] == 's27') or (row[0] == 's3') or (row[0] == 's13') or (row[0] == 's23'):
+            rawData.append(row)
+
+
+# Initially process Points            
+pointsData = [] # Raw data without sid & known; x & y stored as Point class
+for row in rawData:
+    k = (len(row)-2)/2  # Number of Points in current processing Row
+    rowPoints = []  
+    for i in range(2, k+2):
+        x=float(row[i])
+        y=float(row[i+k])
+        rowPoints.append(Point(x,y))
+    pointsData.append(rowPoints)
+
+pointsDataCopy = copy.deepcopy(pointsData) #To be used in the SECOND SETTING
+#FIRST SETTING:
+# Choose duration-threshold to be 50ms & dispersion-threshold to be 2 degree
+# Data generated at 1000Hz, hence the moving window spans about 50 points 
+SC = 50 # spanning capacity: Number of Points that moving window initially span
+DT = 2*math.sqrt(97**2+56**2) # dispersion-threshold
```

Comment:
magic numbers

Reply:
1 degree is approximately 97 units horizontally and 56 units vertically. So I convert 2 degree to that DT value.
Chosen duration-threshold is 50ms. Data generated as 1000Hz => generated 1 point/1ms.
So each moving window should contains 50 points. 
Not sure about these.

Reply:
No need to calculate the norm here, just `DT = 97+56` is ok.

Reply:
`H_UNIT_SIZE = 97`
`V_UNIT_SIZE = 56`